debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/ToastedReplicationMessageColumn.java

```diff
@@ -18,6 +18,10 @@
  */
 public class ToastedReplicationMessageColumn extends AbstractReplicationMessageColumn {
 
+    public static enum ToastedValue {
```

Comment:
What's the purpose of this enum with a single value? It's a bit unclear.

Reply:
@gunnarmorling This one is used in value converters. When detected it is replaced with the placeholder value depending on target datatype - either string or binary

Reply:
Ok, got it. Why is it not a simple constant then:

    public static final Object UNCHANGED_TOAST_MARKER = new Object();

And in the code above you could ask `if data == UNCHANGED_TOAST_MARKER ...`. Using an enum implies that there are multiple values really.

Reply:
@gunnarmorling Done